internal/tsc_wrapped/tsconfig.json

```diff
@@ -10,8 +10,9 @@
       "es5",
       "es2015.core",
       "es2015.collection",
+      "es2015.promise",
+      // TODO: remove "es2015.iterable" this and replace all instances of for .. of with for
```

Comment:
Here is a formatted version of this comment.
// TODO: Remove "es2015.iterable" by replacing all instances of "for .. of".

I think you should also track this TODO within a github issue where it is explained more throughouly. As things read right now, I'm not sure who replacing for of loops will let you replace that particular lib

Reply:
Added an explainer in the PR description. 
Ill fix up this issue if we go through with this PR.